Casks/futubull.rb

```diff
@@ -11,10 +14,10 @@
   livecheck do
     url "https://www.futunn.com/download/history?client=11"
     strategy :page_match do |page|
-      match = page.match(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[_-](\d+)[._-]Website.*?\.dmg/i)
+      match = page.match(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[._-]Website.*?\.dmg/i)
       next if match.blank?
 
-      "#{match[1]},#{match[2]}"
+      (match[1]).to_s
     end
   end
```

Comment:
@BleedingSky The livecheck block can be simplified.

```suggestion
  livecheck do
    url "https://www.futunn.com/download/history?client=11"
    regex(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[._-]Website\.dmg/i)
  end
```